charts/trino/templates/configmap-coordinator.yaml

```diff
@@ -46,7 +46,6 @@ data:
     http-server.http.port={{ .Values.service.port }}
     query.max-memory={{ .Values.server.config.query.maxMemory }}
     query.max-memory-per-node={{ .Values.server.config.query.maxMemoryPerNode }}
-    query.max-total-memory-per-node={{ .Values.server.config.query.maxTotalMemoryPerNode }}
```

Comment:
Can we make this conditional on the version of Trino being used?

Reply:
Agreed, let's have charts backward compatible with previous Trino versions.

Reply:
Agreed -- I've updated the PR accordingly.